libvips/colour/icc_transform.c

```diff
@@ -785,7 +789,7 @@ vips_icc_class_init(VipsIccClass *class)
 static void
 vips_icc_init(VipsIcc *icc)
 {
-	icc->intent = VIPS_INTENT_RELATIVE;
+	icc->intent = VIPS_INTENT_AUTO;
```

Comment:
Won't changing the default break a lot of old code?

Reply:
How about picking the user's requested rendering intent, and if that's not available, falling back to the profile's suggested intent? Then only raising an error if both are missing.

We wouldn't need the AUTO setting, and libvips behaviour with old code would not change.

Reply:
Good idea! I've incorporated this in the latest changeset and also targeted the [`8.16`](https://github.com/libvips/libvips/tree/8.16) branch instead.